### ISSUE TITLE
Docs should mention where Redash user state gets stored

### ISSUE THREAD

**Issue by jeffwidman** [getredash/redash#875]
Docs should mention where user state (accounts/alert subscriptions, etc) gets stored. 

Looking at code it looks like this is a dedicated postgres instance, but I'm not 100% sure.

**Comment by arikfr** [comment]
Yes, Re:dash uses a dedicated Postgres database to store metadata (users, queries, query results, etc) and Redis as a Celery broker and for temporary state. In #856 we expand the usage of Redis to store tasks state.

**Comment by JyothiGandi** [comment]
Hi Arik, 
I'm using Redash 4 for my use cases which handles confidential information. So I would like to know in detail about what it is storing exactly in Redis. Particularly I have 2 questions.

1) Does Redis contain "query results"? Let me know even if it is temporary.
2) What is that "temporary state" you mentioned in above comment?

**Comment by arikfr** [comment]
The current state is the same as mentioned in my above comment. Redis does not store query results.

If you need more details, you're welcome to review the code: that's the benefit of an open source project. :-) Alternatively you're welcome to engage in a support contract, you can contact us via team@redash.io.

Thanks.

**Comment by JyothiGandi** [comment]
Thanks a lot, this helps
